src/test/database/fakeListRemote.spec.ts

```diff
@@ -0,0 +1,103 @@
+import * as pathLib from "path";
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+
+import { ListRemote } from "../../database/listRemote";
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
```

Comment:
This `chai.use(chaiAsPromised)` shouldn't be required as it's done in the [mocha bootstrap file](src/test/helpers/mocha-bootstrap.js)